Multi-org membership (tech preview) (#5351)
elastic/docs-content: deploy-manage/cloud-organization/manage-multiple-organizations.md

Although you can access multiple organizations from the same {{ecloud}} account, each organization is independent. Each organization has its own set of resources, users, settings, and billing and licensing. Because of this, you need to be logged in to the organization you want to manage to make changes to its resources and settings, or invite users to join it.

You can perform the following tasks to manage multiple organizations:

**Admin actions:**

* [Create a new organization](#create-a-new-organization)
* [Invite users to join additional organizations](#invite-users-to-join-additional-organizations)
* [View your users' organization memberships](#view-your-users-organization-memberships)

**User actions:**

* [Accept an invitation](#accept-an-invitation)
* [Log in with multiple organizations](#log-in-with-multiple-organizations)
* [View the organizations you have access to](#view-organizations)
* [Switch to a different organization](#switch-to-a-different-organization)
* [Leave an organization](#leave-an-organization)

## Technical preview limitations

The following limitations apply during the multi-organization tech preview:

> * **Marketplace organizations:** Users who belong to an organization with a [marketplace subscription](/deploy-manage/deploy/elastic-cloud/subscribe-from-marketplace.md) can't join other organizations, and users from other organizations can't join marketplace organizations.

Reviewer: ```suggestion

```

Reviewer: Hoping to remove this limitation in the days to come

> 
> * **Marketplace organizations:** Users who belong to an organization with a [marketplace subscription](/deploy-manage/deploy/elastic-cloud/subscribe-from-marketplace.md) can't join other organizations, and users from other organizations can't join marketplace organizations.
> * **Email notification links:** Organization-specific links in notification emails, such as billing or deployment health alerts, don't yet carry organization context. If you belong to multiple organizations, an email link might take you to the wrong organization. Switch to the appropriate organization before taking action.

Author: ```suggestion
The following limitations apply during the multi-organization tech preview:

* **Email notification links:** Organization-specific links in notification emails, such as billing or deployment health alerts, don't yet carry organization context. If you belong to multiple organizations, an email link might take you to the wrong organization. Switch to the appropriate organization before taking action.
```

---
applies_to:
  deployment:
    ess: preview
  serverless: preview
products:
  - id: cloud-hosted
  - id: cloud-serverless
navigation_title: Manage multiple organizations
---

# Manage multiple {{ecloud}} organizations

> You can create or access multiple organizations from a single {{ecloud}} account. You might want to use multiple organizations for reasons such as the following:

Reviewer 2: Should we add a tiny description of what organizations are (conceptually)? 
I tried to look up a suitable page to link to, but the best description I found is that it `is the umbrella for all of your Elastic Cloud resources, users, and account settings` on the [Serverless](https://www.elastic.co/docs/deploy-manage/deploy/elastic-cloud/serverless) page.

I know this page is aimed at experienced (admin) users, who we'd expect to be familiar with the concept, but no harm to explain what orgs are for those that need a refresher or landed on the wrong page.

## Create a new organization

You can create a new organization at any time. Each organization starts with its own [14-day trial](/deploy-manage/deploy/elastic-cloud/create-an-organization.md#general-sign-up-trial-what-is-included-in-my-trial).

To create a new organization:

1. Log in to the [{{ecloud}} Console](https://cloud.elastic.co?page=docs&placement=docs-body).
2. From the top navigation menu, click on the user menu and select **Profile**.
3. Click the **My organizations** tab.
4. From the **Organizations** page, click {icon}`plus_in_circle` **Create organization**.
5. Enter an optional name for your organization, and then click **Create organization**.

After you create the organization, you can switch to it by clicking the organization name in the **Organizations** list.

:::{tip}
You can also create a new organization by clicking on your current organization name and selecting {icon}`plus_in_circle`  **Create**.
:::

:::{include} _snippets/view-orgs.md
:::

:::{include} _snippets/switch-orgs.md
:::

% tech preview content below - this would probably be split off onto other pages at GA time

## Invite users to join additional organizations

> Because users are managed at the organization level, you must [invite users](/deploy-manage/users-roles/cloud-organization/manage-users.md#ec-invite-users) from within the organization that you want them to join. You can't invite users to join multiple organizations at once.

Reviewer 2: Not sure if this is clearer, but here's my attempt at it:

```suggestion
To join an organization, a user must receive an invitation sent by a current member of that organization. [Invite users](/deploy-manage/users-roles/cloud-organization/manage-users.md#ec-invite-users) from within the desired organization to add them as members. You can't invite users to join multiple organizations at once.
```

Author: only org owners can invite users - what about this current phrasing do you find confusing?

Reviewer 2: It's mostly this bit `Because users are managed at the organization level` (as opposed to project level?) I'd just chuck it.  We're also in a section titled `Invite users to join additional organizations` so this seems unnecessary.

It's totally up to you.

Author: good idea 🫡

If a user already has an {{ecloud}} account, then they don't need to sign up again. Instead, they can log in with their selected login method. 

If your organization uses [SAML SSO](/deploy-manage/users-roles/cloud-organization/configure-saml-authentication.md), then you don't need to invite users to join the organization. Users are added to the organization automatically when they log in to your identity provider SSO URL.

Organizations can have different authentication requirements. For example, one organization might enforce SAML SSO, while another organization might not enforce any specific login method. If your organization enforces a specific login method, then the user will need to use that method to log in, and might be prompted to re-authenticate. 

> 

Reviewer: ```suggestion
## Accept or decline an invitation

When you're invited to join an organization, you receive an email with a link to accept the invitation. Invitations expire after 72 hours.

To accept an invitation:

1. Open the invitation email and click **Accept invitation**.
2. Log in if prompted. If you already have an active session, the invitation is accepted in your current browser.
3. After accepting, you are switched to the new organization automatically.

:::{note}
If the organization enforces a specific login method (such as SAML SSO or social login), you will be redirected to that login flow when accepting the invitation.
:::

To decline an invitation, you can ignore the email. The invitation expires automatically after 72 hours. You can also decline it from the **My organizations** tab, where pending invitations are listed.
```

Author: pending invitations are not present on the **my organizations** screen in QA

Reviewer: @csierra is there something missing in QA or?

> ## View your users' organization memberships

Reviewer: ```suggestion
## Leave an organization

You can leave an organization at any time, as long as you are a member of at least one other organization. If you are the only owner of an organization, you must transfer ownership before leaving.

To leave an organization:

1. Log in to the [{{ecloud}} Console](https://cloud.elastic.co?page=docs&placement=docs-body).
2. From the top navigation menu, click on the user menu and select **Profile**.
3. Click the **My organizations** tab.
4. Find the organization you want to leave and click **Leave organization**.
5. Confirm that you want to leave.

:::{note}
Leaving an organization revokes your access to all of its resources, including deployments, projects, and settings. This action cannot be undone — you will need a new invitation to rejoin.
:::

## View your users' organization memberships
```

* [View your users' organization memberships](#view-your-users-organization-memberships)
% * [Delete an organization](#delete-an-organization)

## Create a new organization

You can create a new organization at any time. Each organization starts with its own [14-day trial](/deploy-manage/deploy/elastic-cloud/create-an-organization.md#general-sign-up-trial-what-is-included-in-my-trial).

To create a new organization:

1. Log in to the [{{ecloud}} Console](https://cloud.elastic.co?page=docs&placement=docs-body).
2. From the top navigation menu, click on the user menu and select **Profile**.
3. Click the **My organizations** tab.
4. From the **Organizations** page, click {icon}`plus_in_circle` **Create organization**.
5. Enter an optional name for your organization, and then click **Create organization**.

After you create the organization, you can switch to it by clicking the organization name in the **Organizations** list.

:::{tip}
You can also create a new organization by clicking on your current organization name and selecting {icon}`plus_in_circle`  **Create**.
:::

:::{include} _snippets/view-orgs.md
:::

:::{include} _snippets/switch-orgs.md
:::

% ## Delete an organization

> % Can it be done? Do we want to document this?

Reviewer: This is not something that can be done by the user. They can only leave an org and we have guardrails for the last Org owner to leave the org (bills paid and resources terminated).

@lloydmeta to double-check
